packages-exp/auth-compat-exp/src/persistence.ts

```diff
@@ -113,16 +113,14 @@ export function _getPersistenceFromRedirect(
   const persistence = win.sessionStorage.getItem(key);
 
   switch (persistence) {
-    case exp.inMemoryPersistence.type:
-      return exp.inMemoryPersistence;
-    case exp.indexedDBLocalPersistence.type:
-      return exp.indexedDBLocalPersistence;
-    case exp.browserSessionPersistence.type:
-      return exp.browserSessionPersistence;
-    case exp.browserLocalPersistence.type:
```

Comment:
Previously this case can never be reached since `exp.indexedDBLocalPersistence.type` is also `LOCAL`. I've changed this switch statement to use `Persistence.xxx` to make it more obvious.

Reply:
Makes sense